pokedex/network/net_groups.lua

```diff
@@ -0,0 +1,98 @@
+local md5 = require "utils.md5"
+local netcore = require "pokedex.network.netcore"
+local settings = require "pokedex.settings"
+local broadcast = require "utils.broadcast"
+
+local net_groups_settings = nil
+
+local M = {}
+
+M.MSG_GROUPS_CHANGED = hash("net_groups_groups_changed")
+
+local function generate_id()
+	local m = md5.new()
+	m:update(tostring(socket.gettime()))
+	return md5.tohex(m:finish()):sub(1, 8)
+end
```

Comment:
This is also done in `pokemon.lua` right? Maybe we should move it into `utils.` somewhere

Reply:
It's profiles.lua, and yes it's similar (not 100% the same, has a 5 instead of an 8).

#494